src/Network/Network.Test/ScenarioTests/PrivateEndpointTests.ps1

```diff
@@ -0,0 +1,101 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+function Check-CmdletReturnType
+{
+    param($cmdletName, $cmdletReturn)
+
+    $cmdletData = Get-Command $cmdletName;
+    Assert-NotNull $cmdletData;
+    [array]$cmdletReturnTypes = $cmdletData.OutputType.Name | Foreach-Object { return ($_ -replace "Microsoft.Azure.Commands.Network.Models.","") };
+    [array]$cmdletReturnTypes = $cmdletReturnTypes | Foreach-Object { return ($_ -replace "System.","") };
+    $realReturnType = $cmdletReturn.GetType().Name -replace "Microsoft.Azure.Commands.Network.Models.","";
+    return $cmdletReturnTypes -contains $realReturnType;
+}
+
+<#
+.SYNOPSIS
+Test creating new PrivateEndpoint 
+#>
+function Test-PrivateEndpointCRUD
+{
+    # Setup
+    $rgname = Get-ResourceGroupName;
+    $rglocation = Get-ProviderLocation ResourceManagement;
+    $rname = Get-ResourceName;
+    $location = Get-ProviderLocation "Microsoft.Network/privateEndpoints";
+    # Dependency parameters
+    $SubnetName = "SubnetName";
+    $SubnetAddressPrefix = "10.0.1.0/24";
+    $PrivateLinkServiceConnectionName = "PrivateLinkServiceConnectionName";
+
+    try
+    {
+        $resourceGroup = New-AzResourceGroup -Name $rgname -Location $rglocation;
+
+        # Create required dependencies
+        $Subnet = New-AzVirtualNetworkSubnetConfig -Name $SubnetName -AddressPrefix $SubnetAddressPrefix;
```

Comment:
Need to create VirtualNetwork referring this Subnet as most child resources are created on client side only.

Reply:
Already changed the case, thanks.